### PR TITLE
Add RBAC roles to device apis

### PR DESCRIPTION
**Pull request by franqq**
device-delete-registeredusers
device-get
device-list-registeredusers
device-list
device-post-registeredusers

### CODE REVIEW

File: api-reference/beta/api/device-get.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | Device.Read.All, Device.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All |
 
+The calling user must also be in one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference): *Global Administrator*, *Users*,
```
Comment (FaithOmbongi) [inline api-reference/beta/api/device-get.md:30]: When it's more than 3 items, use the list format to improve scanning.

File: api-reference/beta/api/device-list-registeredusers.md [hunk 2]
```diff
@@ -28,6 +28,9 @@ One of the following permissions is required to call this API. To learn more, in
 
 [!INCLUDE [limited-info](../../includes/limited-info.md)]
 
+The calling user must also be in one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference): *Global Administrator*, *Users*,
```
Comment (FaithOmbongi) [inline api-reference/beta/api/device-list-registeredusers.md:31]: Use the list format to improve scanning

File: api-reference/beta/api/device-list.md [hunk 3]
```diff
@@ -26,6 +26,9 @@ One of the following permissions is required to call this API. To learn more, in
 | Delegated (personal Microsoft account) | Not supported. |
 | Application | Device.Read.All, Device.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All |
 
+The calling user must also be in one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference): *Global Administrator*, *Users*,
```
Comment (FaithOmbongi) [inline api-reference/beta/api/device-list.md:29]: List format

File: api-reference/beta/api/device-list-registeredusers.md (hunk 2 above)
Comment (franqq) [reply api-reference/beta/api/device-list-registeredusers.md]: fixed

**Review by FaithOmbongi — APPROVED** [review]

**Pull request merged**